src/modules/healthcheck/healthcheck.controller.ts

```diff
@@ -22,7 +22,7 @@ export class HealthcheckController {
     private readonly mem: MemoryHealthIndicator,
   ) {}
 
-  @Get('ready')
+  @Get('API micro-service health check performs dependent databases connection and memory heap check')
```

Comment:
The `Get` decorator is used to create a path i.e. /api/v1/ready
This from above looks like the description of it

Reply:
Ah apologies I'll double check I was planning to update the description

Reply:
Have rectified.